Packs/AzureNetworkSecurityGroups/Integrations/AzureNetworkSecurityGroups/README.md

```diff
@@ -323,11 +307,11 @@ Update a security rule. If one does not exist, it will be created.
 | direction | The direction of the rule. Possible values are: "Inbound" and "Outbound". Possible values are: Inbound, Outbound. | Optional | 
 | action | Whether to allow the traffic. Possible values are "Allow" and "Deny". Possible values are: Allow, Deny. | Optional | 
 | protocol | The protocol on which to apply the rule. Possible values are: "Any", "TCP", "UDP", and "ICMP". Possible values are: Any, TCP, UDP, ICMP. | Optional | 
-| source | The source IP address range from which incoming traffic will be allowed or denied by this rule. Possible values are "Any", an IP address range, an application security group, or a default tag. Default is "Any". | Optional | 
-| priority | The priority by which the rules will be processed. The lower the number, the higher the priority. We recommend leaving gaps between rules - 100, 200, 300, etc. - so that it is easier to add new rules without having to edit existing rules. Default is "4096". | Optional | 
```

Comment:
Are we taking out these "default is x" lines on purpose? Looks like it theyre missing since they werent in the yml. Please check if these defaults make sense according to the api docs, and if they do then lets add them to the yml